src/android/NetworkManager.java

```diff
@@ -46,17 +54,17 @@ public class NetworkManager extends CordovaPlugin {
 
     public static final String WIFI = "wifi";
     public static final String WIMAX = "wimax";
-    // mobile
+  // mobile
```

Comment:
```suggestion
    // mobile
```
Just a nitpick, but can we change the indention level back to 4 spaces?

I prefer a 2-space indention, but Cordova uses a 4-space indention everywhere.